### PR TITLE
IGNITE-13446

### PR DESCRIPTION
**Pull request by Sega76**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [x] There is a single JIRA ticket related to the pull request. 
- [x] The web-link to the pull request is attached to the JIRA ticket.
- [x] The JIRA ticket has the _Patch Available_ state.
- [x] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [x] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [x] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/ducktests/tests/ignitetest/services/utils/templates/log4j.xml.j2 [hunk 1]
```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE log4j:configuration PUBLIC "-//APACHE//DTD LOG4J 1.2//EN"
-    "http://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/xml/doc-files/log4j.dtd">
 
-<!--
+{#
```
Comment (anton-vinogradov) [inline modules/ducktests/tests/ignitetest/services/utils/templates/log4j.xml.j2:3]: Is this related to the issue?

Comment (Sega76) [reply modules/ducktests/tests/ignitetest/services/utils/templates/log4j.xml.j2]: fixed

File: modules/ducktests/tests/ignitetest/services/utils/config_template.py [hunk 2]
```diff
@@ -43,7 +44,10 @@ def render(self, **kwargs):
         Render configuration.
         """
         kwargs.update(self.default_params)
-        res = self.template.render(**kwargs)
+        unfiltered = self.template.render(**kwargs)
+
+        res = '\n'.join(list(filter(lambda line: re.search(r'\S', line), unfiltered.split('\n'))))
```
Comment (ivandasch) [inline modules/ducktests/tests/ignitetest/services/utils/config_template.py:49]: join accepts iterator, there is no need to create list

Comment (Sega76) [reply modules/ducktests/tests/ignitetest/services/utils/config_template.py]: thanks, done

**Review by ivandasch — APPROVED** [review]
